docs/en/Community-Articles/2023-01-30-How-To-Contribute-To-a-Open-Source-Software/how-can-I-contribute-to-open-source-projects-on-github.md

```diff
@@ -0,0 +1,105 @@
+
+or alternatively use the `git clone https://github.com/username/abp.git` command
+
+![clone-image](images/clone-image.png)
+
+## Step 3: Create a New Branch
+
+In this step, you need to create a new branch of your own before you start developing it. Open the repo on GitHub Desktop and create a new branch. When creating a new branch, be careful which branch you create it on.
+
+or alternatively use the `git checkout -b new-branch` command
+
+![branch-image](images/branch-image.png)
+
+## Step 4: Development
+
+Choose a suitable IDE to develop on the new branch you created. In order not to complicate things, we will create a `Developers.md` file and process it. Let's enter a sample text in the Developers file.
+
+![developer-list](images/developer-list.png)
+
+As you can see, all changes made to the repo are reflected directly in Github Desktop.
+
+![github-desktop-change](images/github-desktop-change.png)
+
+## Step 5: Commit
+
+The commit operation is used to save the changes you have made. It is useful to commit after certain operations are done in the project. For the changes you will make, it is useful to write a short sentence describing what you've done. Press the `Commit to <branch-name>` button to commit. 
+
+or alternatively use the `git add .` and `git commit -m "new-branch"` command
+
+![commit-image](images/commit-image.png)
+
+## Step 6: Publish the Changes
+
+The changes you have made so far are only visible on your local machine. You need to publish these changes to submit them to your forked repository. Please press the publish branch button to publish. 
+
+or alternatively use the `git push origin new-branch` command
+
+
+![push-image](images/git-push-image.png)
+
+## Step 7: Create a Pull Request
+
+After the push, the pull request `Create Pull Request` button will appear on the GitHub Desktop. Click it and create a pull request.
+
+![github-desktop-pull-request](images/github-desktop-pull-request.png)
+
+You can also make a pull request from the repo in your GitHub profile.
+
+![compare-pull-request](images/pull-request-image.png)
+
+Before requesting a pull request, it is useful to select one of ABP's tags to indicate the change you have made. You can easily filter the bug or enhancement using the ABP tags. Make sure that the branch you created reaches the correct branch. After briefly describing your changes in the title and description, click the `Create pull request` button. This will send a pull request to the original repository. If the pull request is approved and merged by the community, your changes will also appear in the main repository.
+
+![open-pull-request](images/open-pull-request-image.png)
```

Comment:
The users that doesn't have access to the repository, can't see that label section and can't modify labels of PR